asf_welsh_energy_consultation/analysis/produce_plots_and_stats.py

```diff
@@ -280,18 +394,28 @@
         )
 
     # To recreate October 2023 analysis
-    if get_data.get_args().calculate_average_installations:
-        subset_year_a_mean = process_data.mean_installations_per_year(2015, 2021)
-        subset_year_a_median = process_data.median_installations_per_year(2015, 2021)
+    if get_args(
+        project_dir=PROJECT_DIR, supp_data_dir=supp_dir
+    ).calculate_average_installations:
```

Comment:
you've defined `arguments` variable above right? i think you can use it again, instead of calling the function a second time.
```suggestion
    if arguments.calculate_average_installations:
```

Reply:
good spot thank you!